CHANGELOG.md

```diff
@@ -1,28 +1,35 @@
 # Changelog
 
-#### [Unreleased](https://github.com/openfga/cli/compare/v0.7.2...HEAD)
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](https://semver.org/).
+
+## [Unreleased]
+
+## [0.7.3] - 2025-08-12
 
 Added:
 - Support running `fga model test` with multiple files using glob patterns (#423)
 
-#### [0.7.2](https://github.com/openfga/cli/compare/v0.7.1...v0.7.2) (2025-07-11)
+## [0.7.2] - 2025-07-11
```

Comment:
Why are there so many format changes?

Reply:
I see, still would've been better to have them in separate PRs so we can link all the PRs that only update the changelog format as I see not all of our repos use the keep a changelog format.